File: 백엔드/AMN/src/main/java/com/ibkpoc/amn/dto/UserValidationResponse.java
package com.ibkpoc.amn.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserValidationResponse {
    private int userId;
    private boolean isValid;
    private boolean isActive; // 사용자 active 여부
    private boolean isMeetingActive; // 미팅 active 여부
    // Constructor, getters, setters
}

File: 앱/IBK_PoC/app/src/main/java/com/ibkpoc/amn/model/UserValidationResponse.kt
// model/UserValidationResponse.kt (추가)
package com.ibkpoc.amn.model

import com.google.gson.annotations.SerializedName

data class UserValidationResponse(
    @SerializedName("userId") val userId: Int,
    @SerializedName("active") val active: Boolean,
    @SerializedName("valid") val isValid: Boolean,
    @SerializedName("meetingActive") val isMeetingActive: Boolean
)